starter-code/src/functions-and-arrays.js

```diff
@@ -74,6 +181,17 @@ var wordsCount = [
   'disobedience',
   'matter'
 ];
+
+function howManyTimes(wordsCount, countWord){
+  if (wordsCount.length === 0){
+    return false;
+  }
+  for (var i = 0; i< wordsCount.length; i++){
+  let numTimes = (countWord.indexOf(wordsCount[i]))
```

Comment:
best to set your variable within the function outside of loops or statements and then modify them within the loop or statement.